src/NumberButton.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function NumberButton({ handleClick, value }) {
+  return (
+    <button type="button" value={value} onClick={(e) => handleClick(e)}>
```

Comment:
```suggestion
    <button type="button" value={value} onClick={handleClick}>
```
이렇게 작성하는게 가독성 측면에서 좋을거같은데 어떻게 생각하시나요!?

Reply:
넵! event 파라미터를 전달 한다는 것을 명시적으로 넣으면 어떨까 하는 생각으로 작성했습니다. 하지만 추후 가독성을 위해 말씀하신 것처럼  작성하는 것이 제가 봐도 좋은 것 같습니다!